usb/cherryusb/host/host_cdc_msc_hid/FreeRTOSConfig.h

```diff
@@ -0,0 +1,7 @@
+#ifndef FREERTOS_CONFIG_H
+#define FREERTOS_CONFIG_H
+
+// This example uses a common include to avoid repetition
+#include "FreeRTOSConfig_examples_common.h"
```

Comment:
This should probably just include https://github.com/raspberrypi/pico-examples/blob/develop/freertos/FreeRTOSConfig_examples_common.h rather than adding a separate copy of the file to the repo?

Reply:
https://github.com/raspberrypi/pico-examples/tree/master/freertos i use this template, freertos uses FreeRTOSConfig.h not FreeRTOSConfig_examples_common.h, so i don't know why add FreeRTOSConfig_examples_common.h.

Reply:
I think maybe you could just do `#include ../../../freertos/FreeRTOSConfig_examples_common.h` in your `FreeRTOSConfig.h`? (although perhaps there's a "nicer" way of doing that without all the `../` ?) And then you could remove `FreeRTOSConfig_examples_common.h` from this PR. :shrug:

Reply:
How about using FreeRTOSConfig.h only? If configs in FreeRTOSConfig_examples_common.h have conflict with other demo like less memory, no feature support like sem & mutex disabled, others will be confused.

Reply:
Hmmm I just checked and I hadn't realised that `FreeRTOSConfig_examples_common.h` doesn't use the normal
```c
#ifndef SOME_SETTING
#define SOME_SETTING some_value
#endif
```
pattern. But @peterharperuk understands all of this much better than I do! :laughing: